lambdas/handlers/authoriser_handler.py

```diff
@@ -0,0 +1,268 @@
+"""
+This code has been modified from AWS blueprint. Below is the original license:
+
+Copyright 2015-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License. A copy of the License is located at
+     http://aws.amazon.com/apache2.0/
+or in the "license" file accompanying this file.
+This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+"""
+
+import logging
+import os
+import re
+import time
+
+import boto3
+import botocore.exceptions
+import jwt
+from boto3.dynamodb.conditions import Key
+from enums.permitted_role import PermittedRole
+from services.dynamo_service import DynamoDBService
+from utils.exceptions import AuthorisationException
+
+from utils.get_aws_region import get_aws_region
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def lambda_handler(event, context):
+    user_roles = []
+    ssm_public_key_parameter_name = os.environ["SSM_PARAM_JWT_TOKEN_PUBLIC_KEY"]
```

Comment:
There's no check to see if this env variable exists before it's use, we use `os.getenv` in some lambdas which it would raise a key error however since we're using `environ` here maybe a check to see if `ssm_public_key_parameter_name` has a value and return 500 if nothing